PathCompiler - Reports invalid position by appending 1 to the index instead of adding by 1

I have this invalid jsonpath
```
"$.store.book[* xxxxxxx]"
```

Which gives me this exception, notice the position is reported as `131` which should have been `13 + 1 = 14` instead
```
Caused by: com.jayway.jsonpath.InvalidPathException: Expected wildcard token to end with ']' on position 131
	at com.jayway.jsonpath.internal.path.PathCompiler.readWildCardToken(PathCompiler.java:487)
	at com.jayway.jsonpath.internal.path.PathCompiler.readNextToken(PathCompiler.java:140)
```

--- a/json-path/src/main/java/com/jayway/jsonpath/internal/path/PathCompiler.java
+++ b/json-path/src/main/java/com/jayway/jsonpath/internal/path/PathCompiler.java
@@ -484,7 +484,8 @@ public class PathCompiler {
         if (inBracket) {
             int wildCardIndex = path.indexOfNextSignificantChar(WILDCARD);
             if (!path.nextSignificantCharIs(wildCardIndex, CLOSE_SQUARE_BRACKET)) {
-                throw new InvalidPathException("Expected wildcard token to end with ']' on position " + wildCardIndex + 1);
+                int offset = wildCardIndex + 1;
+                throw new InvalidPathException("Expected wildcard token to end with ']' on position " + offset);
             }
             int bracketCloseIndex = path.indexOfNextSignificantChar(wildCardIndex, CLOSE_SQUARE_BRACKET);
             path.setPosition(bracketCloseIndex + 1);

